src/builders/typescript/typescript-direct-builder.ts

```diff
@@ -0,0 +1,186 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as ts from "typescript";
+import * as Linter from "tslint";
+import { glob } from "multi-glob";
+import { ILinterOptionsRaw, LintResult } from "tslint/lib/lint";
+import Configuration from "../../configuration/configuration";
+import { Logger } from "../../utils/logger";
+
+interface TsConfig {
+    compilerOptions: any;
```

Comment:
It's not  `ts.CompilerOptions`. That is why [L29](https://github.com/QuatroCode/simplr-gulp/pull/40/files/cee3801b1d9e879b092d792c5886b9a2e6c14abc#diff-9675c861591be1df0ef841ed36d9875cR29) is needed:
```ts
// Parse compiler options (e.g. jsx: react -> jsx: 2)
let parsedCompilerOptions = ts.convertCompilerOptionsFromJson(tsConfigFromJson.compilerOptions, process.cwd());``